core/trino-main/src/main/java/io/trino/operator/scalar/StringFunctions.java

```diff
@@ -203,6 +203,15 @@ public static Slice reverse(@SqlType("varchar(x)") Slice slice)
         return SliceUtf8.reverse(slice);
     }
 
+    @Description("Reverse all code points in a given string")
+    @ScalarFunction(value = "reverse")
+    @LiteralParameters("x")
+    @SqlType("varchar(x)")
+    public static Slice charReverse(@LiteralParameter("x") Long x, @SqlType("char(x)") Slice slice)
```

Comment:
Use primitive `long` for the argument.

Reply:
I see that some functions use **Long**, while others use **long**. This should be standardized here

Reply:
Yeah, that's a mistake in the other functions. The value can never be null for a `char(n)` type, so `long` is sufficient.

Reply:
Why return is varchar(x)? Maybe char(x)?

Reply:
@martint @Heltman Thank you for pointing out the opinions. I have made the necessary modifications. Please confirm if there are any other opinions

Reply:
LGTM